samples/EFT/Parsec.Samples.EFT/Program.cs

```diff
@@ -14,6 +16,14 @@ static void Main(string[] args)
 
             eft.Read();
             eft.ExportJson(fileName + ".json");
+
+            // Import the created json file
+            var eftFromJson = Deserializer.ReadFromJson<EFT>(fileName + ".json");
+
+            string name = System.IO.Path.GetFileNameWithoutExtension(fileName);
```

Comment:
There's a property called `FileNameWithoutExtension` in all classes that inherit from `FileBase`, so here you can use:
```cs
eftFromJson.Write(eftFromJson.FileNameWithoutExtension + ".modified.EFT");
```

Reply:
the output file is named .modified.EFT after this change is made. both of these print empty strings to the console.

``Console.WriteLine(eftFromJson.FileName);``
``Console.WriteLine(eftFromJson.FileNameWithoutExtension);``

Reply:
Good catch, that means json imports need fixing. I'll merge this as it is and modify it once I fix the filename for json imported objects